include/zephyr/drivers/fuel_gauge.h

```diff
@@ -37,45 +37,53 @@ extern "C" {
 #define FUEL_GAUGE_AVG_CURRENT 0
 
 /** Battery current (uA); negative=discharging */
-#define FUEL_GAUGE_CURRENT		FUEL_GAUGE_AVG_CURRENT + 1
+#define FUEL_GAUGE_CURRENT		    FUEL_GAUGE_AVG_CURRENT + 1
```

Comment:
Was this from a `clang-format`? If so, please have that as a separate commit.

Reply:
Separated in new revision